aea/crypto/ethereum.py

```diff
@@ -93,7 +93,9 @@ def load_private_key_from_path(cls, file_name) -> Account:
         path = Path(file_name)
         with open(path, "r") as key:
             data = key.read()
-            account = Account.from_key(data)
+            account = Account.from_key(  # pylint: disable=no-value-for-parameter
```

Comment:
The methods here can be used both as class and instance methods; but pylint does not detect that